app/guid-file/template.hbs

```diff
@@ -110,14 +110,14 @@
             {{#if (or (eq show 'edit') (eq show 'view_edit'))}}
                 {{#if isEditableFile}}
                     {{#if canEdit}}
-                        <div class='panel panel-default EditPanel {{if (and (eq show "view_edit") canEdit) "col-sm-6"}}'>
-                            {{#if (await fileText)}}
+                        {{#if (not (eq (await fileText) null))}}
```

Comment:
I'm sure this won't be the only place where we are bit by not strictly checking null.